fraqbot/Local/newjob.py

```diff
@@ -0,0 +1,84 @@
+import logging
+import os
+import random
+import sys
+
+from Legobot.Lego import Lego
+
+
+logger = logging.getLogger(__name__)
+LOCAL_DIR = os.path.join(os.path.abspath(os.path.dirname(__file__)))
+
+if LOCAL_DIR not in sys.path:
+    sys.path.append(LOCAL_DIR)
+
+
+import helpers as h  # noqa E402
+
+
+class NewJob(Lego):
+    def __init__(self, baseplate, lock, *args, **kwargs):
+        super().__init__(baseplate, lock, acl=kwargs.get('acl'))
+        self.role_modifiers_list = []
+        self.roles_list = []
+        self.companies_list = []
+        role_modifiers = h.load_file(
+            os.path.join(LOCAL_DIR, 'lists', 'role_modifiers.txt'), raw=True)
+        roles = h.load_file(
+            os.path.join(LOCAL_DIR, 'lists', 'roles.txt'), raw=True)
+        companies = h.load_file(
+            os.path.join(LOCAL_DIR, 'lists', 'companies.txt'), raw=True)
+
+        if role_modifiers and roles and companies :
+            self.role_modifiers_list = role_modifiers.splitlines()
+            self.roles_list = roles.splitlines()
+            self.companies_list = companies.splitlines()
+
+    def listening_for(self, message):
+        text = message.get('text')
+
+        return isinstance(text, str) and (text.startswith('!job') or text.startswith('!newjob'))
```

Comment:
This condition needs some parentheses: condition 1 and (condition 2 or condition 3)

Reply:
I think I pushed this change alread...